homeassistant/components/huawei_lte/sensor.py

```diff
@@ -424,23 +424,9 @@ class SensorMeta(NamedTuple):
         formatter=lambda x: ({"0": "Auto", "1": "Manual"}.get(x, "Unknown"), None),
         entity_category=EntityCategory.CONFIG,
     ),
+    # TODO: check if this is still needed, the pretty-formated netmode was converted to select
```

Comment:
As discussed, let's keep the sensor. We could convert the formatter keys to `NetworkModeEnum` values here though, like in the select.